API.md

```diff
@@ -1444,6 +1444,87 @@ this._viewer.setPropertiesForAnnotation('Pdftron', 1, {
 });
 ```
 
+#### getAnnotationAtPoint
+Gets the annotation at the (x, y) position in screen coordinates.
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+x | integer | the x-coordinate of the point
+y | integer | the y-coordinate of the point
+distanceThreshold | double | maximum distance from the point (x, y) to the annotation for it to be considered a hit (in dp)
+minimumLineWeight | double | For very thin lines, it is almost impossible to hit the actual line. This specifies a minimum line thickness (in screen coordinates) for the purpose of calculating whether a point is inside the annotation or not (in dp)
+
+Returns a Promise.
+
+Promise Parameters:
+
+Name | Type | Description
+--- | --- | ---
+annotation | object | the annotation found in the format of `{id: string, pageNumber: number, type: string, rect: {x1: number, y1: number, x2: number, y2: number}}`
+
+```js
+this._viewer.getAnnotationAtPoint(167, 287, 100, 10).then((annotation) => {
+  if (annotation) {
+    console.log('Annotation found at point (167, 287) has id:', annotation.id);
+  }
+})
+```
+
+#### getAnnotationsAtLine
```

Comment:
bad API name, it should be getAnnotationAt (one) and getAnnotationListAt (multiple), it could be a line, a rect etc.